src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (`name: ${name}, chip version: ${version}, wheels: ${wheels}`);
+    },
```

Comment:
You should get name / version / wheels not from function's parameters, but from object. Use 'this.' (Use concatenation to avoid linter error for long string).
```suggestion
    get info() {
      return (`name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`);
    },
```

Reply:
Thanks for the advice with concatenation! I had trouble with how to display it in one line and avoid long string